frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -157,6 +248,30 @@ function ExpressionEditorSuggestionsListItem({
   );
 }
 
+function Footer({
+  suggestion,
+  highlightedIndex,
+}: {
+  suggestion: WithIndex<SuggestionFooter>;
+  highlightedIndex: number;
+}) {
+  function handleMouseDownCapture(evt: MouseEvent) {
+    // prevent the dropdown from closing
+    evt.preventDefault();
```

Comment:
This prevents the link from being draggable

Reply:
Still unresolved. I don't think it should prevent this PR from being merged, but let's keep it somewhere in the backlog as a P3 bug.